feat(FDICDC-49): Add Debezium Server Generic Template docs (#135)
mia-platform/documentation: docs/fast_data/connectors/debezium_cdc.md

</details>

### Debezium Server Generic Template

> If none of the databases mentioned in the previous sections are being used, it is still possible to connect any custom database with the Debezium Server.

Reviewer: I wouldn't say any custom databases but the ones supported by debezium that we don't provide as a service in the mkpl

> In this case, the Debezium Server Generic template can be used, which is more flexible and allows for a tailor-made configuration. It is possible to configure it with any sink configurations supported by Debezium as outlined in the [official documentation](https://debezium.io/documentation/reference/stable/connectors/index.html). The template configuration requires setting up the database user, permission, and the JMX Prometheus Exporter configuration.

Reviewer: What do you mean with `The template configuration requires setting up the database user, permission`? Are you referring to the application.properties needed for debezium server? In that case I wouldn't put it like that but just say `The template configuration requires setting up the application.properties as indicated in the official docs`or something like that

Author: Changed.
`By referring to the [official documentation](https://debezium.io/documentation/reference/stable/connectors/index.html), you can configure any sink configurations supported by Debezium. To set up the template configuration, you must configure the *application.properties* file as directed in the documentation. Additionally, the JMX Prometheus Exporter configuration must also be configured.`

> In this case, the Debezium Server Generic template can be used, which is more flexible and allows for a tailor-made configuration. It is possible to configure it with any sink configurations supported by Debezium as outlined in the [official documentation](https://debezium.io/documentation/reference/stable/connectors/index.html). The template configuration requires setting up the database user, permission, and the JMX Prometheus Exporter configuration.

Reviewer: ```suggestion
In this case, the Debezium Server Generic template can be used since it is more flexible and allows a tailor-made configuration. It is possible to configure it with any sink configurations supported by Debezium as outlined in the [official documentation](https://debezium.io/documentation/reference/stable/connectors/index.html). The template configuration requires setting up the database user, permission, and the JMX Prometheus Exporter configuration.
```
